BAZProject/RemoteData/MovieAPI.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  MovieAPI.swift
+//  BAZProject
+//
+//
+
+import Foundation
+
+class MovieAPI {
+
+    private var movies: [ResultMovie] = []
+
+    func getMovies(completion: @escaping(Result<[ResultMovie],Error>) -> Void)  {
+        let remoteData = LoadRemotedata()
```

Comment:
Permite que el remoteData pueda ser inyectado mediante un inicializador/constructor.
```suggestion
    class MovieAPI {
      private let remoteData: LoadRemoteData
      
      init(remoteData: LoadRemoteData) {
        self.remoteData = remoteData
      }
    }
```